pycbc/strain/strain.py

```diff
@@ -1975,9 +1977,12 @@ def from_cli(cls, ifo, args, maxlen):
             idq_state_channel = ':'.join([ifo, args.idq_state_channel[ifo]])
 
         if args.frame_type:
-            frame_src = pycbc.frame.frame_paths(args.frame_type[ifo],
-                                                args.start_time,
-                                                args.end_time)
+            frame_src = pycbc.frame.frame_paths(
+                args.frame_type[ifo],
+                args.start_time,
+                args.end_time,
+                site=ifo[0]
```

Comment:
I think (unlike channel type) this would not work with the "lying" case of `--frame-type H1:L:L_H00` or `H1:L_RAW`. Does this matter?